test/e2e_node/docker_util.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e_node
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/blang/semver"
+	"github.com/docker/docker/client"
+)
+
+// getDockerAPIVersion returns the Docker's API version.
+func getDockerAPIVersion() (semver.Version, error) {
+	c, err := client.NewClient("unix:///var/run/docker.sock", "", nil, nil)
```

Comment:
I think it's a good idea to have a separate file to host all docker specific util code, so we can make changes in one place in the future.

There are other tests (e.g., garbage_collector_test.go, security_context.go ,etc) that connects to docker directly to check the version or more. Could you define the variable in this file, and change other tests use this?